clio-dataaccess-model/src/main/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/ElasticsearchIndex.scala

```diff
@@ -26,8 +26,9 @@ abstract class ElasticsearchIndex[Document] {
     * at this moment.
```

Comment:
Is our source-of-truth storage strategy described somewhere? I looked in confluence but didn't see it.

There is https://github.com/broadinstitute/tbl/blob/master/clio/es.org#what-source-of-truth-should-we-maintain-to-rebuild-clio but it's more of a design doc and not a spec for what is implemented.